packages/next/src/build/webpack/config/blocks/css/index.ts

```diff
@@ -619,7 +619,7 @@ export const css = curry(async function css(
           insert: function (linkTag: HTMLLinkElement) {
             if (typeof _N_E_STYLE_LOAD === 'function') {
               const { href, onload, onerror } = linkTag
-              _N_E_STYLE_LOAD(new URL(href).pathname).then(
+              _N_E_STYLE_LOAD(new URL(href).toString()).then(
```

Comment:
Hi @sokra, is there any blocker to revert to the full href instead of the pathname for the asset URL? 
You made the change in the commit [77e6b03](https://github.com/vercel/next.js/commit/77e6b03) of the PR #66500.

Reply:
It probably needs path name for SSR. During SSR we don't have the full href.

But you could try to change it and check the test cases

Reply:
This change is the only one impacting the runtime code, all the others are related to tests using App Router and `next/dynamic`.

Moreover, it's included in a `isClient` condition so it wouldn't impact the SSR.

Reply:
Hello @noreiller  and @sokra .

This fix seems really required as assetsPrefix is breaking in such cases. What can be best way to fix this?

Reply:
Hi @sokra, what can I do more than the current fix? The issue blocks my team from upgrading to next@15.

Reply:
Sounds good to me

Reply:
```suggestion
              _N_E_STYLE_LOAD(href).then(
```

Reply:
@sokra Thanks, the suggestion is applied.

Reply:
Hello @sokra  . Any update on such changes?